config/_shared.json

```diff
@@ -181,5 +182,6 @@
 	"twemoji_cdn_url": "https://s0.wp.com/wp-content/mu-plugins/wpcom-smileys/twemoji/2/",
 	"woocommerce_blog_id": 113771570,
 	"wpcom_signup_id": false,
-	"wpcom_signup_key": false
+	"wpcom_signup_key": false,
+	"max_request_time_logs_per_second": 50
```

Comment:
Done. I changed it to `statsd_analytics_response_time_max_logs_per_second`. Let me know if that's too verbose... 😃